docker/launch-hostagent.sh

```diff
@@ -14,10 +14,11 @@ KUBECONFIG=/usr/local/etc/kubeconfig
 if [ -w /mnt/cni-bin ]; then
     # Install CNI plugin binary
     mkdir -p ${CNIBIN}
-    rm -f ${CNIBIN}/*
 if [ -z != $CHAINED_MODE ] && [ "$CHAINED_MODE" == "true" ]; then
```

Comment:
Trying to understand why this if condition was not met with the previous change.

Reply:
This condition is for chained-mode setups. The issue is with rke setups where loopback binary will be present in CNIBIN folder and since we were deleting all binaries inside CNIBIN, loopback binary was also getting deleted (and not added back).

Reply:
Oh ok. I did not realize there were other binaries in there. Who copies them there ?

Reply:
Initcontainer (cnideploy) container of hostagent pod copies it https://github.com/noironetworks/aci-containers/blob/master/docker/launch-cnideploy.sh

Reply:
Thanks

Reply:
The idea is to delete the specific binary, not everything. It applies in both conditions, L19 as well as L21. We would hit L19 in non-ACI-CNI case.